contracts/tansu/src/tests/test_dao.rs

```diff
@@ -0,0 +1,345 @@
+use super::test_utils::{create_test_data, init_contract};
+use crate::{
+    errors::ContractErrors,
+    types::{Badge, Dao, ProposalStatus, PublicVote, Vote, VoteChoice},
+};
+use soroban_sdk::testutils::{Address as _, Ledger, arbitrary::std::println};
+use soroban_sdk::{Address, String, vec};
+
+#[test]
+fn proposal_flow() {
```

Comment:
Duplicate right? I thought now you wanted to have all these estimates in a single file?